playbooks/roles/mysql/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+dependencies:
+  - name: nginx-proxy
+    when: MYSQL_EXPORTER_ENABLED
+    # The username is a bit misleading – should actually be "prometheus", since it's the server
+    # authenticating, and the same username is used for scraping other exporters, but it's not worth
+    # the hassle to change it.
+    NGINX_PROXY_USERNAME: node-exporter
+    NGINX_PROXY_PASSWORD: "{{ MYSQL_EXPORTER_PASSWORD }}"
+    NGINX_PROXY_AUTH_DOMAIN: Prometheus MySQL Exporter
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/mysql-exporter
+    NGINX_PROXY_PUBLIC_PORT: 19104
+    NGINX_PROXY_INTERNAL_PORT: 9104
```

Comment:
Similar comment as above, we actually don't see us changing this port for any reason. In fact changing it would also mean changing the public port to match it but with a prefixed "1", and it would also mean changing the security group on every server using this to open ingress on the proxy public port, which is just way too much work. So for that reason we simply don't bother making it a variable.

Reply:
Note that it's the same case for MongoDB/HAProxy -- the ports are not variables, for all the same reasons, even though they are reused.